composer.json

```diff
@@ -30,6 +30,7 @@
         "symfony/config": "2.8.*",
         "symfony/dependency-injection": "2.8.*",
         "symfony/event-dispatcher": "2.8.*",
+        "symfony/filesystem": "2.8.*",
```

Comment:
@robertdown ,
To avoid merge conflict pain, recommend just bringing this composer package into main codebase now.